<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPR">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPR">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of  Red Player</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPB">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPB">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of Blue player</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="fileRadio">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>file configuration</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="putRadio">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>put piece one by one</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
